db/schema.rb

```diff
@@ -408,6 +408,34 @@
     t.index ["job_user_id"], name: "index_invoices_on_job_user_id_uniq", unique: true
   end
 
+  create_table "job_digest_occupations", force: :cascade do |t|
+    t.bigint "job_digest_id"
+    t.bigint "occupation_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["job_digest_id"], name: "index_job_digest_occupations_on_job_digest_id"
+    t.index ["occupation_id"], name: "index_job_digest_occupations_on_occupation_id"
+  end
+
+  create_table "job_digest_subscribers", force: :cascade do |t|
+    t.string "email"
+    t.string "uuid", limit: 36
+    t.bigint "user_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["user_id"], name: "index_job_digest_subscribers_on_user_id"
+    t.index ["uuid"], name: "index_job_digest_subscribers_on_uuid", unique: true
+  end
+
+  create_table "job_digests", force: :cascade do |t|
+    t.string "city"
+    t.integer "notification_frequency"
+    t.bigint "job_digest_subscriber_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["job_digest_subscriber_id"], name: "index_job_digests_on_job_digest_subscriber_id"
+  end
+
```

Comment:
Oh, totally forgot to explain this. Moving between branches doesn't reset/rollback the database changes in another branch causing this file to be updated even though the changes belong to another branch.. 

There are a few ways to go about this:

1. Make sure to rollback `rails db:rollback STEP=1` each migration that you've performed (step being the number of migrations)
2. Don't commit `db/schema.rb` when changing unrelated stuff in another branch
  - Can easy for forget (i.e `git add .`)
  - Needs manual attention/edits if you do other database changes in the new branch, plus if you restart the server or start a console the `db/schema.rb` file will be automatically overwritten.
3. Maintain an, optional, per branch database
  - Requires a custom script (works by manipulating local git config options)
  - Requires you to remember to invoke that script before running `rails db:migrate` in the new branch
  - Just Works™ when you actually remember and its then easy to move back and forth branches
  - I've written such a script before, albeit for MySQL, but it should only require minor changes to work for posgres ([see this gist](https://gist.github.com/buren/6961940))

Lets take a look at that tomorrow @bex1 :)